test/store.test.js

```diff
@@ -335,9 +337,11 @@ describe('Using Mori', function () {
 });
 
 describe('Using Immutable', function () {
-  var Immutable = require('immutable');
+  var Immutable = require('immutable'),
+      ImmutableProxy = require('../lib/collections/ImmutableProxy');
```

Comment:
Can we do one var declaration at a time? It's repetitive but makes refactoring easier.